notebooks/Generating+SDSS+Source+Table.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
Looks like this notebook needs some markdown at the top to explain what is going on. Every notebook needs a title, a short introduction that lists the goals of the notebook ("In this notebook we will produce x, y and z"), and then a "Requirements" section that tells the user what they need to install for the notebook to run (beyond what comes with the package via the `pip install requirements.txt` command run in the `setup.py` script).

Reply:
Actually in general this notebook is very low on explanatory markdown... Is it one of yours, @jiwoncpark ? Is it supposed to be checked in or not?

Reply:
Not supposed to be checked in... it was an experimental notebook. Deleted.